templates/default/mgm-server-start.sh.erb

```diff
@@ -23,11 +23,20 @@ if [ ! -e <%= node['mysql']['base_dir'] %>/bin/ndb_mgmd ] ; then
     exit 3
 fi
 
+INITIAL_START_ARG=
+if [ -n "$INITIAL_START" ]; then
+    INITIAL_START_ARG="--initial"
+fi
+
+SERVICE_ARG=
+if [ -n "$SERVICE_NAME" ]; then
+    SERVICE_ARG="--service-name $SERVICE_NAME"
```

Comment:
same here, baking an on-premise setup will fail since there is no `$SERVICE_NAME` 
I think the default value should be `ndb_mgmd` as defined in the pid name?

Reply:
Default can be the binary name for sure